src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
I'm bit on fence between casting vs using explicit typed reference. So it'll be basically
```js
const self: Observable<T> = this;
self.doxxx();
```

vs. 

```js
((Observable<T>)this).doXXX()...
```

correct? While I like prior snippet's clarity of type to `self`, also makes feel does it necessarily have to hold another reference which is repeated over every operator. (course I also does agree @masaeedu about casting, just speaking out loud to hear some opinions)